virttest/guest_agent.py

```diff
@@ -877,6 +877,46 @@ def guest_exec_status(self, pid):
         cmd = "guest-exec-status"
         return self._cmd_args_update(cmd, pid=pid)
 
+    def ssh_add_authorized_keys(self, username, *keys, reset=None):
+        """
+        Add ssh public keys into guest-aent.
+
+        :param username: usrename of host
+        :param keys: value of public keys
+        :return: command handle
+        """
+        cmd = "guest-ssh-add-authorized-keys"
+        if reset:
+            reset = True
+        else:
+            reset = False
```

Comment:
Would you help demonstrate why we have this conversion instead of directly have `reset` as a boolean type value?

Reply:
I thought it wrong, forget it.